classes/PluginInstall.class.php

```diff
@@ -33,7 +33,8 @@ public function scripts($hook)
 		if ($hook !== $this->page) {
 			return;
 		}
-		wp_enqueue_script('classicpress-directory-integration-js-plugin', plugins_url('../scripts/plugin-page.js', __FILE__), ['jquery'], false, true);
+		wp_enqueue_script( 'classicpress-directory-integration-js-plugin', plugins_url( '../scripts/plugin-page.js', __FILE__ ), array( 'wp-i18n' ), false, true );
+		wp_set_script_translations( 'classicpress-directory-integration-js', 'classicpress-directory-integration', plugin_dir_path( 'classicpress-directory-integration' ) . 'languages' );
```

Comment:
I think you have to change the handle also here:
`wp_set_script_translations( 'classicpress-directory-integration-js-plugin', ....`

Reply:
Good spot! Now done.